tests/test_mixer.py

```diff
@@ -1,6 +1,8 @@
 import unittest
 from unittest import mock
 
+import copy
```

Comment:
Copy is from the standard library, so it should be in the same block as `unittest`.

```suggestion
import copy
import unittest
from unittest import mock
```